File: CH_6/example6-1textclock/src/main/java/com/example/example6_1textclock/MainActivity.java
package com.example.example6_1textclock;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextClock;

public class MainActivity extends AppCompatActivity {

    Button GetTime;
    TextClock clock;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        GetTime = findViewById(R.id.getTime);
        clock = findViewById(R.id.clock1);
        clock.setVisibility(View.INVISIBLE);

        GetTime.setOnClickListener(v -> clock.setVisibility(View.VISIBLE));
    }
}

File: Chapter10/ex02_practice10_2/src/main/java/com/example/ex02_practice10_2/MainActivity.kt
package com.example.ex02_practice10_2

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ImageView
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    var imageViews = arrayOf<ImageView>(iv1,iv2,iv3,iv4,iv5,iv6,iv7,iv8,iv9)


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        for (i in 0..imageViews.size){
            imageViews[i].setOnClickListener{

            }
        }
    }
}
